### PR TITLE
Prepare for Release 1.2.7

### PR DESCRIPTION
**Pull request by ggjjj**
Prepare for Release 1.2.7 with version update with changelog.

## Azure IoT Edge PR checklist:

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines and Best Practices
- [x] I have read the [contribution guidelines](https://github.com/azure/iotedge#contributing).
- [x] Title of the pull request is clear and informative.
- [x] Description of the pull request includes a concise summary of the enhancement or bug fix.

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.
- Description of the pull request includes 
	- [ ] concise summary of tests added/modified
	- [ ] local testing done.  

### Draft PRs
- Open the PR in `Draft` mode if it is:
	- Work in progress or not intended to be merged.
	- Encountering multiple pipeline failures and working on fixes.

_Note: We use the kodiakhq bot to merge PRs once the necessary checks and approvals are in place. When it merges a PR, kodiakhq converts the PR title to the commit title, PR description to the commit description, and squashes all the commits in the PR to a single commit. The net effect is that entire PR becomes a single commit. Please follow the best practices mentioned [here](https://chris.beams.io/posts/git-commit/#:~:text=The%20seven%20rules%20of%20a%20great%20Git%20commit,what%20and%20why%20vs.%20how%20For%20example%3A%20) for the PR title and description_

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+# 1.2.7 (2021-01-19)
+## aziot-edge
+### Bug Fixes
+* Update vulnerable nix version ( [ca6958f](https://github.com/Azure/iotedge/commit/ca6958f7a3995c43973e4fbd006c1be737b60fe8))
+* New IoTedge check called proxy-settings which verifies proxy settings ([4983128](https://github.com/Azure/iotedge/commit/49831285a02de9189ac338237aaa0f529a72c559))
+* Removed Moby check ([27a14d8](https://github.com/Azure/iotedge/commit/27a14d817d8de78b562691945689fa4400de56b6))
+* Fix for workload socket issue for concurrent module creation ([5712dcc](https://github.com/Azure/iotedge/commit/5712dcc28498121d890082d5c884d9855cc40efd))
+* Addition of device ID to edge CA common name to support large number of devices ([6627c7a](https://github.com/Azure/iotedge/commit/6627c7a835ed6b252da00faa29b9d9b8e5cd501b))
+
```
Comment (yophilav) [inline CHANGELOG.md:9]: Nit: Double newline between release.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
+# 1.2.7 (2021-01-19)
+## aziot-edge
+### Bug Fixes
+* Update vulnerable nix version ( [ca6958f](https://github.com/Azure/iotedge/commit/ca6958f7a3995c43973e4fbd006c1be737b60fe8))
+* New IoTedge check called proxy-settings which verifies proxy settings ([4983128](https://github.com/Azure/iotedge/commit/49831285a02de9189ac338237aaa0f529a72c559))
+* Removed Moby check ([27a14d8](https://github.com/Azure/iotedge/commit/27a14d817d8de78b562691945689fa4400de56b6))
+* Fix for workload socket issue for concurrent module creation ([5712dcc](https://github.com/Azure/iotedge/commit/5712dcc28498121d890082d5c884d9855cc40efd))
+* Addition of device ID to edge CA common name to support large number of devices ([6627c7a](https://github.com/Azure/iotedge/commit/6627c7a835ed6b252da00faa29b9d9b8e5cd501b))
```
Comment (yophilav) [inline CHANGELOG.md:8]: nit: white space inside the enclosing ()

File: edgelet/version.txt [hunk 3]
```diff
@@ -1 +1 @@
-1.2.5
+1.2.7
```
Comment (yophilav) [inline edgelet/version.txt:1]: Nice! We always have the edgelet version the same as the edgeAgent & edgeHub version (even if that means skipping a version number).

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,12 @@
+# 1.2.7 (2021-01-19)
```
Comment (yophilav) [inline CHANGELOG.md:1]: Don't forget the upcoming merge for edgeAgent & edgeHub image update for the security patch.

**Review by yophilav — DISMISSED** [review]

**Review by damonbarry — DISMISSED** [review]
Didn't have a chance to look carefully at release notes, but we can always update that later if we find a problem. Otherwise looks good.

**Review by yophilav — APPROVED** [review]
Danke! 😄

**Pull request merged**